server/src/interfaces/error-handler-interface.ts

```diff
@@ -1,6 +1,9 @@
+
 export interface PortalErrorData {
   message: string;
   status: number;
-  errorMsg?: any;
+  type: string;
+  // eslint-disable-next-line
+  stacktrace?: any;
```

Comment:
```suggestion
  stacktrace?: unknown;
```
Aber das sollte doch ein string[] sein oder nicht?